handler.go

```diff
@@ -32,6 +32,10 @@ type cleanupCommand struct {
 	steps []func() error
 }
 
+var (
+	ErrorHub429 = errors.New("Hub replied with a 429 error code")
```

Comment:
Minor naming improvement: I think it's better to use a common well-known http status name instead of code, like ErrHubTooManyRequests

Reply:
Good idea. I'll do the same for corresponding changes in frontman shortly.

Fixed.